docs/en/user_guides/analysis_tools.md

```diff
@@ -1 +1,147 @@
 # Analysis tools
+
+- [Analysis tools](#analysis-tools)
+  - [Count number of parameters](#count-number-of-parameters)
+  - [Publish a model](#publish-a-model)
+  - [Reproducibility](#reproducibility)
+  - [Log Analysis](#log-analysis)
+  - [Visualize Datasets](#visualize-datasets)
+  - [MAE Visualization](#mae-visualization)
+  - [Use t-SNE](#use-t-sne)
+
+## Count number of parameters
+
+```shell
+python tools/analysis_tools/count_parameters.py ${CONFIG_FILE}
+```
+
+An example:
+
+```shell
+python tools/analysis_tools/count_parameters.py configs/selfsup/mocov2/mocov2_resnet50_8xb32-coslr-200e_in1k.py
+```
+
+## Publish a model
+
+Before you publish a model, you may want to
+
+- Convert model weights to CPU tensors.
+- Delete the optimizer states.
+- Compute the hash of the checkpoint file and append the hash id to the filename.
+
+```shell
+python tools/model_converters/publish_model.py ${INPUT_FILENAME} ${OUTPUT_FILENAME}
+```
+
+An example:
+
+```shell
+python tools/model_converters/publish_model.py YOUR/PATH/epoch_100.pth YOUR/PATH/epoch_100_output.pth
+```
+
+## Reproducibility
+
+If you want to make your performance exactly reproducible, please switch on `--deterministic` to train the final model to be published. Note that this flag will switch off `torch.backends.cudnn.benchmark` and slow down the training speed.
```

Comment:
currently we use `--cfg-options randomness.deterministic=True` to switch on `deterministic`